rosbag2_tests/test/memory_management.cpp

```diff
@@ -41,9 +42,7 @@ TestMemoryManagement::get_initialized_serialized_message(size_t capacity)
         int error = rcutils_char_array_fini(msg);
         delete msg;
         if (error != RCUTILS_RET_OK) {
-          RCUTILS_LOG_ERROR_NAMED(
-            "rosbag2_storage",
-            "Leaking memory. Error: %s", rcutils_get_error_string_safe());
+          std::cout << "Leaking memory" << std::endl;
```

Comment:
is there not ROS_ERROR logging functionality yet in ROS2? should this be a ``//TODO`` to reinstate once there is?

Reply:
Do you have a link to more info about what its being replaced with?

Reply:
I think this page covers it all right now (assuming you mean what's replacing `ROS_INFO`):

https://github.com/ros2/ros2/wiki/Logging

The major difference is that since there's no singleton node in ROS 2 (can have zero to many nodes in a process) the logging macros don't know which node to associate the logging with by default. You can still make "named" logging calls which allowing to name them with a string rather than using the node name, and there's a normal unnamed log call that just uses `""` for the name, but we don't recommend that.

Reply:
Note that we use these logging macros outside test code everywhere. But we should probably not change this here, I agree.